docs: update exercises to match v0.4.0 release (#7)
bomctl/bomctl-playground: README.md

Software Bill of Materials (SBOMs) are imported into the cache, manipulated, and then exported.

### Format Agnostic

> Generating SBOMs for complex systems usually involves [Multiple SBOM Files](#multiple-sbom-files), and sometimes you have no control over the format of SBOMs provided for sub-components. Internally, `bomctl` parses and stores the SBOMs in a way that is agnostic to any of the formats we support for importing and exporting. This means you can import SBOMs in different formats and easily perform operations on them without having to first convert them and worry about losing information in the process. In addition, this allow for easy conversion of provided SBOMs and easy generation of SBOMs for your system in all the support formats so your users can take what they need.

Reviewer: - natural casing for link text
- line length (`bomctl` is enforced at 120 or less, I think `markdownlint` defaults to 80 or less)
- recommend rephrasing `you` statements
- couple of typos

```suggestion
Generating SBOMs for complex systems usually involves [multiple SBOM files](#multiple-sbom-files), and sometimes the
format of SBOMs provided for sub-components is outside of a user's control. Internally, `bomctl` parses and stores
SBOMs in a format-agnostic way. `bomctl` can import SBOMs in various formats and easily perform operations on them
without having to first be converted, potentially losing information in the process. In addition, this allows for
easy conversion of provided SBOMs and generation of SBOMs in all supported formats.
```

1. __Encoding__: Controls what encoding the resulting document will be, either json or xml. Default: `json`  
    __Note__ that your format choice changes the encoding options.
1. __Format__: Controls the format and format version. Default: `cyclonedx`  
   __Note__ the generic `spdx` and `cyclonedx` format options. These will use the latest version of that format supported by `bomctl`.
            There are more explicit format options for each format version if an older version is desired.

#### __Step 2: Export to a file__

Export the an SBOM to a local directory.

We're going to export the cdx SBOM fetched in the first [exercise](#step-1-fetch). __If you haven't already, go back and fetch it.__

``` bash
bomctl export urn:uuid:f360ad8b-dc41-4256-afed-337a04dff5db -f spdx -o test.spdx.json
```

Note that we previously fetched a cdx SBOM and are exporting an SPDX SBOM.

#### __Step 3: Export to another tool__

Grype is a tool that can scan SBOMs for vulnerabilities and accepts them as input from stdin. bomctl can facilitate this.

```bash
bomctl export urn:uuid:f360ad8b-dc41-4256-afed-337a04dff5db | grype
```

> ### Merge Sboms

Reviewer: ```suggestion
### Merge SBOMs
```

The Merge command will merge two or more documents that exist in local storage. This will leave the original SBOMs unaltered in  
the cache and will generate a new SBOM with the combined, deduplicated contents.

More info regarding the Merge command and merging strategy can be found in the help menu for the command:

```bash
bomctl merge --help
```

Next lets get some simple test boms in the db to illustrate how it works.

``` bash
bomctl import examples/bomctl_merge_A.cdx.json examples/bomctl_merge_B.cdx.json
```

> View them in the db if you like:

Reviewer: ```suggestion
View the imported SBOMs in the database:
```

``` bash
bomctl list
```

These are intentionally small, manageable SBOMs. So you can cat out the contents if you like, or simply open them in the editor to view.  
Once you're ready to merge:

```bash
bomctl merge urn:uuid:f360ad8b-dc41-4256-afed-337a04dff5db urn:uuid:3de02d44-f9c6-4a94-bf48-eb92730dc3b5
```

You can add a `--name` or `-n` flag to give the merged SBOM a specific name, otherwise a UUID will be generated and used as the name.  
Do another list to see the new third SBOM that has been generated, while both previous SBOMs are still available.

``` bash
bomctl list
```

> Lets export the merged bom so we can see the before and after:

Reviewer: ```suggestion
Let's export the merged SBOM so we can see the before and after:
```
